lib/js/atoms/Tile/Tile.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { createLocalVue, shallowMount } from '@vue/test-utils';
+
+import Tile from './Tile.vue';
+import { ICONS } from '../Icon';
+const defaultProps = {
+	text: '',
+	eyebrowText: '',
+	interactive: true,
+	additionalText: '',
+	leftIcon: null,
+	rightIcon: null,
+};
+const createComponent = function ({
+	text,
+	eyebrowText,
+	interactive,
+	additionalText,
+	leftIcon,
+	rightIcon,
+}) {
+	const localVue = createLocalVue();
+	return shallowMount(Tile, {
+		localVue,
+		mocks: {},
+		propsData: { interactive, additionalText, leftIcon, rightIcon },
+		slots: {
+			eyebrowText,
+			default: text,
+		},
+		stubs: {},
+	});
+};
+
+describe('Tile', () => {
+	it('Is interactive without left icon', () => {
+		const props = {
+			text: 'text text',
+			eyebrowText: 'eyebrowText text',
+			additionalText: 'additionalText text',
+			rightIcon: ICONS.FA_ANGLE_RIGHT,
+		};
+		const component = createComponent({ ...defaultProps, ...props });
```

Comment:
you can set default values in `createComponent()` method

Reply:
Nope. Linter throws errors.

Reply:
You need to know how to talk with them :)
https://github.com/bethinkpl/design-system/pull/103/files#diff-6812c905c4c4fd5574700c076c2a2f6d74cf66935ba2ec312bf102fbde421b62R15-R22